markdown/ranger/ranger-kerberos.html.md.erb

```diff
@@ -0,0 +1,209 @@
+
+To configure Ranger access to a HAWQ cluster enabled with Kerberos user authentication, you must:
+
+- Identify an existing HAWQ administrative role or create a new HAWQ administrative role for Ranger lookup of HAWQ resources
+- Create a Kerberos principal for the lookup role
+- Update the Ranger HAWQ service definition
+
+### Procedure <a id="kerb_ra2hawq_proc"></a>
+
+Perform the following procedure to enable the Ranger Administration Host to look up resources in your kerberized HAWQ cluster. You will perform operations on the HAWQ \<master\>, \<ranger-admin-node\>, and \<kdc-server\> nodes.
+
+1. Log in to the HAWQ master node and set up your environment:
+
+    ``` shell
+    $ ssh gpadmin@<master>
+    gpadmin@master$ . /usr/local/hawq/greenplum_path.sh
+    ```
+
+2. Identify an existing or create a new HAWQ administrative role for Ranger resource lookup. For example, to create a new administrative role:
+
+    ``` shell
+    gpadmin@master$ psql -c 'CREATE ROLE "rangerlookup_hawq" with LOGIN SUPERUSER;' 
+    ```
+   
+    You may choose a different name for the Ranger lookup role.
+
+3. Log in to the KDC server system and generate a principal for the HAWQ `rangerlookup_hawq` role. Substitute your Kerberos \<realm\>. For example:
+
+    ``` shell
+    $ ssh root@<kdc-server>
+    root@kdc-server$ kadmin.local -q "addprinc -pw changeme rangerlookup_hawq@REALM.DOMAIN"
+    ```
+    
+    You do not need to generate a keytab file for the `rangerlookup_hawq` principal because you will provide the password in the HAWQ service definition of the Ranger Admin UI.
+
+4. Start the Ranger Admin UI in a supported web browser. The default URL is \<ranger-admin-node\>:6080. 
+
+5. Locate the HAWQ service definition and press the **Edit** button. 
+
+6. Update the applicable **Config Properties** fields:
+
+    **HAWQ User Name*** - The HAWQ Ranger lookup role you identified or created in Step 2 above.  
+    **HAWQ User Password*** - The password you assigned to the HAWQ Ranger lookup role principal.  
+    **HAWQ Kerberos Service Name** - `postgres`  
+
+7. Click the **Test Connection** button. If you connect successfully, Ranger displays a dialog with the message:
+
+    ``` pre
+    Connected Successfully.
+    ```
+
+    You successfully configured Ranger to use Kerberos authentication when looking up HAWQ resource names.
+
+8. **Save** your changes to permanently configure Kerberos authentication for HAWQ Ranger resource lookup.
+
+
+## Configure HAWQ for Kerberized Ranger<a id="rps2ra_kerb_cfg"></a>
+
+When Ranger Authorization is enabled for HAWQ, the Ranger Plug-in Service fetches policies from the Ranger Administration host. When Kerberos is enabled for Ranger, you must specifically configure the Ranger Plug-in Service for this communication.
```

Comment:
"When Kerberos is enabled for Ranger" -> "**If** Kerberos is enabled for Ranger"